examples/amp-story/analytics.html

```diff
@@ -79,11 +89,13 @@
                 }
               },
               "trackBookendEnter": {
-                "on": "story-bookend-enter",
+                "on": "story-open",
+                "tagName": "amp-story-bookend",
```

Comment:
If I want to collect all `story-open` events from all elements. Can I not have the `tagName` in the config, but including the `${tagName}` in the outgoing request instead?

Reply:
This will probably not work, since some elements don't necessarily have a mapping for their real tagName and their analytics tagName (e.g. the info dialog is a `div`, but we can use an alternative tagName for analytics called `amp-story-info-dialog`).

I think a solution here is what @gmajoulet proposes in another comment and refactor these so that their real tagName matches with their analytics tagName.

Reply:
We don't current support `${tagName}` now. So it's possible for us to always replace `${tagName}` with the analytics tagName without worrying about the actual tagName. 

I'm proposing that because I think it will make the configuration easier. One can simply put
```
'trackStoryEnter': {
  "request": "https://some-example?eventType=${tagName}"
  "on": "story-open"
}
```
for all story-open requests. 

This is not blocking the current PR : )

Reply:
Got it, that makes sense. I have filed #25892 to address this separately.